fdmi/plugins/sched.c

```diff
@@ -1,31 +1,48 @@
 /* -*- C -*- */
 /*
-* Copyright (c) 2021 Seagate Technology LLC and/or its Affiliates
-*
-* Licensed under the Apache License, Version 2.0 (the "License");
-* you may not use this file except in compliance with the License.
-* You may obtain a copy of the License at
-*
-*    http://www.apache.org/licenses/LICENSE-2.0
-*
-* Unless required by applicable law or agreed to in writing, software
-* distributed under the License is distributed on an "AS IS" BASIS,
-* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-* See the License for the specific language governing permissions and
-* limitations under the License.
-*
-* For any questions about this software or licensing,
-* please email opensource@seagate.com or cortx-questions@seagate.com.
-*
-*/
+ * Copyright (c) 2021 Seagate Technology LLC and/or its Affiliates
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * For any questions about this software or licensing,
+ * please email opensource@seagate.com or cortx-questions@seagate.com.
+ *
+ */
 
 #define M0_TRACE_SUBSYSTEM M0_TRACE_SUBSYS_FDMI
 
+#include <unistd.h>
+#include <fcntl.h>
 #include "fdmi/plugins/sched.h"
-/*
+
+#define FDMI_PLUGIN_EP "/etc/fdmi_plugin_ep"
```

Comment:
Can we remove `FDMI_PLUGIN_EP`? not used anymore?

Reply:
yes, it was a debug.